WcaOnRails/app/models/competition.rb

```diff
@@ -699,15 +699,15 @@ def psych_sheet_event(event)
 
     prev_registration = nil
     registrations.each_with_index do |registration, i|
-      if event.recommended_format.sort_by == :single
+      if sort_by == 'single'
         rank = registration.single_rank
         prev_rank = prev_registration&.single_rank
       else
         rank = registration.average_rank
         prev_rank = prev_registration&.average_rank
       end
-      registration.tied_previous = (rank == prev_rank)
       break if !rank # hasn't competed in this event yet and all subsequent registrations too
+      registration.tied_previous = (rank == prev_rank)
```

Comment:
I didn't really need to do this. But I thought it was a more appropriate place to break. Once users no longer have a rank, we stop giving them a position, so we should also stop seeing if they are tied.